CMSgov/mdct-mcr
# TFWS ETL In Deploy script

## Description
Adding the ETL to the deploy script, also fixing a bug in update.ts

### How to test
Deploy code
check DynamoDB, see that seeded data uses old format
execute
`aws lambda invoke /dev/null --endpoint-url http://localhost:3003 --function-name database-local-s3Etl`
view DynamoDB, see data has been transformed
Check services/uploads/local-buckets/local-mcpar-forms and validate IDs in dynamo point to actual S3 objects.

### Changed Dependencies
adding KSUID to services/database

## Code author checklist

## Reviewer checklist (two different people)
